tests/integration_tests.rs

```diff
@@ -622,6 +622,32 @@ fn complex_parallel_for_appender_loop() {
     unsafe { weld_value_free(ret_value) };
 }
 
+fn simple_for_vectorizable_loop() {
```

Comment:
I've only implemented `merger` so far but `appender` is coming soon. Right now I'll just print something if it vectorizes (so I know all the merger loops are being vectorized now), but yes, checking the string is better so we don't need to print. I'll add that.